src/app/App.js

```diff
@@ -11,21 +16,18 @@ import withStyles from '@material-ui/core/styles/withStyles';
 
 import HomeIcon from '@material-ui/icons/Home';
 
-import ErrorBoundary from 'atoms/ErrorBoundary';
-import Resume from 'modules/Resume';
-import CreateResumePage from 'pages/CreateResumePage';
-import {ProfileAuth, ProfileStaff} from 'pages/ProfilePage';
-import theme from 'styles/theme';
+import ErrorBoundary from '../lib/atoms/ErrorBoundary';
+import Resume from '../components/Resume/Resume';
 
-import {useAuth0} from 'lib/auth0';
+import {ProfileAuth, ProfileStaff} from '../components/ProfilePage';
+import theme from '../styles/theme';
 
-/* TODO remove these or move to pages folder */
-import Home from 'components/Other/Home.js';
-import Contacts from 'components/Contacts/Contacts.container';
-import SearchContact from 'components/SearchContact/SearchContact';
-import TalentHome from 'components/TalentHome/TalentHome';
+import {useAuth0} from '../lib/Auth0/auth0';
 
-import NavBarIcons from './components/NavigationBar/NavBarIcons';
+import Home from '../components/Home/Home';
```

Comment:
So, we've been inconsistent about how we set up our import paths here, but since we're doing a bunch of work it's probably a good idea to standardize it so it's clean.

I'd recommend always referencing any of the top level folders (e.g. `components`, `state`, `lib`, etc.) without using the `..`, so the above would be `components/Home/Home`. Items which are referencing other files in the same component folder can still use `.` and `..` though